fv3core/Makefile

```diff
@@ -142,7 +142,8 @@ get_test_data:
 	rm -rf $(TEST_DATA_HOST) && \
 	$(MAKE) sync_test_data && \
 	$(MAKE) unpack_test_data ;\
-	fi
+	fi; \
+	echo $(FORTRAN_SERIALIZED_DATA_VERSION)
```

Comment:
You can leave this if you like, pointing it out in case this was meant to be temporary.

Reply:
This was actually intended. It allows scripts to extract the updated version string without duplicating logic.

Reply:
Since printing this version is an important side-effect, please add a comment saying this so it doesn't get deleted by someone who doesn't know.

Reply:
Still need this.

Reply:
I added it above...

Reply:
I do worry about someone not noticing a comment 6 lines away when deleting this, if you can move it down to line 150-153. If not that's fine.